schema.json

```diff
@@ -119,334 +122,223 @@
 					"enum": [
 						"person", "organization", "family", "software"
 					]
-				}
-			}
-		},
-		"subnotes": {
-			"$id": "#/definitions/subnotes",
-			"type": "object",
-			"title": "Subnotes",
-			"required": [
-				"type",
-				"content"
-			],
-			"properties": {
-				"type": {
-					"$id": "#/definitions/subnotes/properties/type",
+				},
+				"uri": {
+					"$id": "#/definitions/agent_refs/properties/uri",
 					"type": "string",
-					"items": {
-						"type": "string",
-						"enum": [
-							"text",
-							"orderedlist",
-							"definedlist"
-						]
-					},
-					"title": "Subnote Type",
-					"minLength": 1,
+					"title": "Agent URI",
+					"description": "Relative link to any agent",
 					"examples": [
-						"definedlist"
+						"agents/23rq4df3asdfaf"
 					],
 					"pattern": "^(.*)$"
-				},
-				"content": {
-					"$id": "#/definitions/subnotes/properties/content",
-					"type": "array",
-					"title": "Subnote Content",
-					"items": {
-						"$id": "#/definitions/subnotes/properties/content/items",
-						"type": "string",
-						"title": "Content Items",
-						"minLength": 1,
-						"examples": [
-							"cupidatat quis culpa amet labore incididunt",
-							"ullamco in Lorem ad minim labore",
-							"laborum dolore adipisicing aliquip incididunt ad",
-							"elit ipsum quis ex do dolore"
-						],
-						"pattern": "^(.*)$"
-					}
 				}
 			}
 		},
-		"notes": {
-			"$id": "#/definitions/notes",
+		"ancestors": {
```

Comment:
Can't we just create a more abstract "reference" definition and then point to that from ancestors, parents, agents and terms?